test/style.js

```diff
@@ -53,3 +53,10 @@ test('vendor prefixed styles are correctly camel-cased', function (t) {
   t.plan(1)
   t.equal(el.style.WebkitTransition, 'opacity 100ms ease')
 })
+
+test('pascal-cased, vendor prefixed styles are not camel-cased', function (t) {
+  var el = mk().node()
+  el.setAttribute('style', 'WebkitTransition: opacity 100ms ease')
```

Comment:
Thanks, Microsoft. Good catch again, the React documentation should be our
source of truth here. It still feels like a regular expression is the way
to go though. To me.
On 26 Feb 2016 11:15 am, "Nicolas Brugneaux" <notifications@github.com>
wrote:

> In test/style.js
> <https://github.com/Olical/react-faux-dom/pull/38#discussion_r54231431>:
>
> > @@ -53,3 +53,10 @@ test('vendor prefixed styles are correctly camel-cased', function (t) {
> >    t.plan(1)
> >    t.equal(el.style.WebkitTransition, 'opacity 100ms ease')
> >  })
> > +
> > +test('pascal-cased, vendor prefixed styles are not camel-cased', function (t) {
> > +  var el = mk().node()
> > +  el.setAttribute('style', 'WebkitTransition: opacity 100ms ease')
>
> I think it's maybe not enough. As specified by React, the vendor prefixes
> *except* ms should be pascal cased. So that'd make sense to check for the
> substring? That use case should be added to the test as well [image:
> :8ball:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Olical/react-faux-dom/pull/38/files#r54231431>.
>